source/manual/alerts/rabbitmq-no-consumers-listening.html.md

```diff
@@ -4,75 +4,106 @@ title: 'RabbitMQ: No consumers listening to queue'
-name should indicate the app responsible for consuming the queue.
+For information about how we use RabbitMQ, see [here][rabbitmq_doc].
 
-The Icinga check is performed by connecting to RabbitMQ's admin API, so the
-information given is from Rabbit's point of view.
+## Criticals
 
-Since we send heartbeats every minute (from each instance of the Publishing
-API) at least one of the consumers for each queue should be active every
-minute. To be conservative, the check checks that at least one consumer of the
-queue is running and has been active within the last 5 minutes.
+### `No consumers listening to queue`
 
-If the check succeeds, it will return the most recent time at which activity
-happened on the queue.
+This check reports a critical error when no consumers are listening to the
+queue.
 
-If the check fails due to no consumers having been active recently, it will
-report a critical failure, giving the most recent idle time, and the idle
-times of each consumer connected to RabbitMQ:
+Icinga connects to RabbitMQ's admin API to check that at least one consumer is
+running for a given RabbitMQ message queue. See
+[here][no_consumers_listening_config] for the alert config.
 
-```
-CRITICAL: No activity for X seconds: idle times are [X, Y, Z]
-```
+### `No activity for X seconds: idle times are [X, Y, Z]`
 
-If the check fails due to there being no consumers connected to the queue, it
-will report a critical failure:
+This checks whether the RabbitMQ consumers for a queue have been active in the
+last 5 minutes.
 
-```
-CRITICAL: "No consumers listening to queue"
-```
+Icinga connects to RabbitMQ's admin API to check on the activity of the
+consumers of a given RabbitMQ message queue. See [here][idle_config] for the
+alert config.
+
+**Note**
+> Publishing API [sends a heartbeat message][heartbeat_messages] every minute to
+> the following queues - `email-alert-service`, `cache_clearing_service-high` and
+> `content_data_api`. These queues receive the heartbeat due to being classed as
+> particularly important. This is configured via the queues bindings matching the
+> routing key used in the [heartbeat][heartbeat]. This should ensure that the
+> consumers are never idle for these queues.
+
+## Troubleshooting
+
+### Check the RabbitMQ logs
+
+* Check the logs for the `rabbitmq` application which can be achieved logging
+  into Kibana and searching for `application: rabbitmq`. Is there evidence of
+  any errors?
+
+### Check the RabbitmQ Grafana dashboard
+
+This [Grafana dashboard][rabbitmq_grafana_dashboard] shows activity across
+multiple exchanges and queues. The main exchange we expect to be monitoring is
+`published_documents` which handles broadcasting to services such as search and
+email-alert-service when content changes across GOV.UK.
+
+Looking at the queue graphs we should look out for the following:
+
+* Check for high 'ready' messages for the queue
+
+This indicates these messages are waiting to be processed in RabbitMQ by the
```

Comment:
Minor: the formatting here means it's ambiguous whether "this" refers to the previous bullet point, or the next one. Can we just tack is on to the end?